underfs/hdfs/pom.xml

```diff
@@ -27,6 +27,7 @@
     <build.path>${project.parent.parent.basedir}/build</build.path>
     <!-- Decouple hadoop versions between north and south bound -->
     <ufs.hadoop.version>2.2.0</ufs.hadoop.version>
+    <lib.jar.name>${project.artifactId}-${ufs.hadoop.version}-${project.version}.jar</lib.jar.name>
```

Comment:
don't we need string "hdfs" in here?

Reply:
project.artifactId already comes with 'alluxio-underfs-hdfs' so that version isn't needed. Currently the jars come out looking like 'alluxio-underfs-hdfs2.2.0-2.0.0-SNAPSHOT.jar'.